erpnext_com/www/docs/user/manual/en/setting-up/contents.md

```diff
@@ -108,6 +108,7 @@ You'll find the global configuration options here, for module-wise setup, visit
 1. [Using Custom Domain On ERPNext](/docs/user/manual/en/setting-up/articles/using-custom-domain-on-erpnext)
 1. [Setup Two Factor Authentication](/docs/user/manual/en/setting-up/articles/setup-two-factor-authentication)
```

Comment:
Set up

Reply:
lol... this was not some this I added... changes it anyways...

Reply:
Though I am never going to get the difference between "set up" and "setup" anytime soon... :(